frontend/src/state/squad/squadReducer.js

```diff
@@ -0,0 +1,97 @@
+import {
+  GET_SQUAD_REQUEST,
+  GET_SQUAD_SUCCESS,
+  GET_SQUAD_ERROR,
+  BROWSE_SQUADS_REQUEST,
+  BROWSE_SQUADS_SUCCESS,
+  BROWSE_SQUADS_ERROR,
+  POST_SQUAD_REQUEST,
+  POST_SQUAD_SUCCESS,
+  POST_SQUAD_ERROR,
+  PUT_SQUAD_REQUEST,
+  PUT_SQUAD_SUCCESS,
+  PUT_SQUAD_ERROR,
+  DELETE_SQUAD_REQUEST,
+  DELETE_SQUAD_SUCCESS,
+  DELETE_SQUAD_ERROR,
+} from "./squadActions";
+
+const initialState = {
+  squad: {},
+  squadList: [],
```

Comment:
I sometimes  default these to `false`, and when I clear them, put them to false, since empty arrays and objects are truthy. I think doing that is a hacky, though. This way definitely more maintainable, as long as we're careful to remember this.

Reply:
Interesting. Yeah, I have always thought of this more like documentation. And then used lodash's `isEmpty` when performing the check haha. Thats an interesting hack though.